src/com/pentalog/pentastagiu/homework2/Client.java

```diff
@@ -0,0 +1,30 @@
+package com.pentalog.pentastagiu.homework2;
+
+public class Client {
+    String FirstName;
```

Comment:
conventia pentru denumirea fieldurilor/metodelor e camelCase: firstName, lastName, depositMoney, withdrawMoney(fara underscore pentru separare)

totodata e recomandat sa ai fielduri private si atunci cand ai nevoie de detaliile respective in alt context(codul din alta clasa) sa generezi getteri sau setteri